students/mathewcm/lesson02/gridPrinter.py

```diff
@@ -0,0 +1,60 @@
+def beam():
+    """
+    beam() function defines one of three beams in a 2x2 grid pattern
+    """
+    print("+----+----+", end=" ")
+
+def post():
+    """
+    post() function defines one of eight posts separating and propping up the beams in a 2x2 grid pattern
+    """
+    print("|     |     |", end=" ")
+
+def printGrid():
```

Comment:
It would have been nice to see some loops or string multiplication here — but you do get to that later — so good!